.github/workflows/clean_environment_tests.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
 
   build-and-test:
     if: github.event_name == 'push' || github.event.pull_request.draft == false
-    uses: Nexus-Mods/NexusMods.App.Meta/.github/workflows/dotnet-build-and-test-with-osx.yaml@ae64a3be780a74e94b59ee463a413083013c8b0c
+    uses: Nexus-Mods/NexusMods.App.Meta/.github/workflows/dotnet-build-and-test-with-osx.yaml@main
```

Comment:
Not using a pinned version of the workflows could cause issues.
Changes to the Meta repo that also require changes to the underlying repos would break CI during the mean time. 
E.g. while the PR with the changes is being reviewed, Ci on all other PRs will break. 

Just pointing it out, in case it others have opinions on this

Reply:
if it causes issues we can always change it back